automated-comments/ruby/two-fer/incorrect_default_param.md

```diff
@@ -0,0 +1 @@
+What could the default value of the paramater be set to in order to avoid having to use a conditional?
```

Comment:
At first glance I was unsure about the question, thinking that it would be posted to the student automatically, and I don't feel like automated comments should encourage an answer.

However, since this is only given to the mentor as a suggestion, I think it's probably fine.

Reply:
@kytrinyx Yeah - I wonder if we should change the rules for disapprove comments now they come from a human. The rules we wrote for approve are to come from computers.